src/installing-from-source.rst

```diff
@@ -28,11 +25,33 @@ Checking dependencies
 ---------------------
 
 You need:
+- `Git <https://git-scm.com/downloads>`_
+- `Python 3.6 or 3.7 <https://www.python.org/downloads/>`_ (3.8+ not supported, see :gh-aw:`this comment <activitywatch/issues/433#issuecomment-653397090>`), check with :code:`python3 -V` (required to build the core components)
+- `Poetry <https://python-poetry.org/docs/#installation>`_, check with :code:`poetry -V` (can be installed like this: :code:`python3 -m pip install poetry`)
+- `Node 12 or higher <https://www.npmjs.com/get-npm>`_, check with :code:`node -v` and :code:`npm -v` (required to build the web UI)
+- `Rust nightly and cargo <https://doc.rust-lang.org/cargo/getting-started/installation.html>`_, nightly can be installed using the following commands:
+
+.. code-block:: sh
+
+   rustup install nightly
+   rustup default nightly
+
+check with :code:`rustc -V` and :code:`cargo -V` (for building aw-server-rust)
+
+**For Windows users:**
+You also need:
+- `gnuwin32 <http://gnuwin32.sourceforge.net/packages/make.htm>`_ that you can use the make command on Windows.
+- `7 Zip <https://www.7-zip.org/>`_
+- `Chocolatey <https://chocolatey.org/docs/installation>`_ for the choco command
+
+Also make sure that you have the following paths are inside the PATH variable:
```

Comment:
Make it clear that these are just the most likely paths, not guaranteed path.

Reply:
Not 100% sure what you mean. Does the problem still exist with the latest commit?

Reply:
No the problem is fixed in the latest commit since it mentions all the packages which should work to run from the cli.